docs/navigation-parameters-reference.md

```diff
@@ -843,6 +843,16 @@ The global search is an input field available in the top navigation bar. The sea
 - **description**: disables the on:keyUp and other internal handlers on the search input field. It is a plain input field then, which can be used to attach your own handlers. If set to `true`, a **searchProvider** must be defined in order to show the search field, which can contain your custom logic. It is recommended to initialize your custom logic in the [**lifeCycle.luigiAfterInit**](lifecycle-hooks.md#luigiafterinit) hook.
 - **since**: 1.5.0
 
+### globalSearchCenteredCancelButton
+- **type**: string
+- **description**: defines the label of the cancel button. It will be displayed if you want to hide the search field on a smaller viewport.
+- **since**: NEXTRELEASE
+
+### searchFieldCentered
+- **type**: boolean
+- **description**: The search input field will be rendered in the center of the shellbar.
+- **since**: NEXTRELEASE
+
```

Comment:
I'd suggest to switch the order. First mention the `searchFieldCentered` and then `globalSearchCenteredCancelButton`. Is `globalSearchCenteredCancelButton` mandatory or optional? What would be shown if it not defined or empty?

Reply:
Since we have to rule for alphabetical order I would leave it as it is.
I would add the optional flag and add a hint that the cancel btn is only availible if searchfield is centered

Reply:
> when a User activates the search via starting typing it has to have a focus state on entire search input not on the buttons on the right.
This works for me.